docs/decisions/app/0006-use-nextjs.md

```diff
@@ -0,0 +1,101 @@
+# Use Next.js for Template
+
+* Status: draft <!-- optional -->
+* Deciders: [list everyone involved in the decision] <!-- optional -->
```

Comment:
We should fill the deciders list in and delete `<!-- optional -->` if we decide to fill in ya? (I think @lorenyu also mentioned this so sorry to just drive by with a similar comment - thanks for working on this 🤓 )